src/components/Apps.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
```

Comment:
Hi, thanks for pointing this out, but since we don't use it in this milestone and its causing linting errors, please put this line about the import to ignore the linting error `// eslint-disable-next-line ` or comment it, let us have stickler greened.

Reply:
code commented